### PR TITLE
Readme overhaul

### PR DESCRIPTION
**Pull request by pavlenex**
I've noticed that our readme has a bit of outdated information and doesn't really link to all relevant places. Since our readme is our GitHub landing page, this PR attempts to organize things and clean things up.

**Preview here** https://github.com/pavlenex/btcpayserver/tree/readme-improvements

Here's what's added:
- Navigation header with links to demo, bug reports, and important places (docs, site, yt API)
- Software screenshot (important to preview how  software looks)
- Adds Developing section, nests (build, run, debug) and adds API section that clarifies our two API's (hopefully)
- Adds new badges (PR welcome, Twitter, Chat, etc) and aligns them properly 
- Simplifies features section
- Adds emoji in headings for more fun :tada: 
 
Here's what's removed:
- YouTube video (we can add a new one once it's done)
- Removes introduction section as it was too wordy, instead adds simple sentence and links to docs
- Removes Azure deploy badge since it'll be added in the azure readme (azure no longer recommended deployment method)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,98 +1,147 @@
...
-* [How to run](#how-to-run)
-* [How to debug](#how-to-debug)
-* [Dependencies](#other-dependencies)
+![BTCPay Server](BTCPayServer/wwwroot/img/btc_pay_BG_twitter.png)
 
-## Features
+<h3 align="center">
+  Accept Bitcoin payments ₿
+</h3>
+<p align="center"> BTCPay Server is a free and open-source Bitcoin payment processor which allows you to accept bitcoin without fees or intermediary.
```
Comment (Kukks) [inline README.md:8]: Should mention self-hosted imo

File: README.md [hunk 2]
```diff
@@ -1,98 +1,147 @@
...
 * Direct, peer-to-peer Bitcoin payments
 * No transaction fees (other than the [network fee](https://en.bitcoin.it/wiki/Miner_fees))
-* No processing fees
-* No middleman
-* No KYC
+* No fees, middleman or KYC
 * Non-custodial (complete control over the private key)
-* Enhanced privacy
-* Enhanced security
+* Enhanced privacy & privacy
```
Comment (Kukks) [inline README.md:95]:
```suggestion
* Enhanced privacy & security
```

File: README.md [hunk 3]
```diff
@@ -1,98 +1,147 @@
...
-* Opt-in [altcoin](https://docs.btcpayserver.org/FAQ/FAQ-Altcoin/) integrations
-* Full compatibility with BitPay API (easy migration)
-* Process payments for others
-* Easy-embeddable Payment buttons
-* Point of sale app
-* Crowdfunding app
-* Payment Requests
-* Internal, full-node reliant wallet with [hardware wallet integration](https://docs.btcpayserver.org/Vault/)
+* Bitcoin-only build
+* Share your instance with friends
```
Comment (Kukks) [inline README.md:101]:
```suggestion
* Share your instance with friends (multi-tenant)
```

File: README.md [hunk 4]
```diff
@@ -1,98 +1,147 @@
...
-## Documentation
+## 📗 Documentation
 
 Please check out our [official website](https://btcpayserver.org/), our [complete documentation](https://docs.btcpayserver.org/) and [FAQ](https://docs.btcpayserver.org/FAQ/) for more details.
 
-If you have trouble using BTCPay, consider joining [communities listed on official website](https://btcpayserver.org/#communityCTA) to get help from BTCPay community members. Only file [Github issue](https://github.com/btcpayserver/btcpayserver/issues) for technical issues you can't resolve through other channels or feature requests you've validated with other members of community.
+If you have trouble using BTCPay, consider joining [communities listed on the official website](https://btcpayserver.org/#communityCTA) to get help from the BTCPay community members. Only file [Github issue](https://github.com/btcpayserver/btcpayserver/issues) for technical issues you can't resolve through other channels or feature requests you've validated with other members of the community.
 
-Main community chat is located on [Mattermost](https://chat.btcpayserver.org/).
+The main community chat is located on [Mattermost](https://chat.btcpayserver.org/).
```
Comment (Kukks) [inline README.md:120]: main or official?

File: README.md [hunk 5]
```diff
@@ -136,23 +185,34 @@ If you need to debug ledger wallet interaction, install the development time cer
...
-## Other dependencies
+### Other dependencies
 
 For more information, see the documentation:
-[How to deploy a BTCPay server instance](https://docs.btcpayserver.org/Deployment/).
+[How to deploy a BTCPay Server instance](https://docs.btcpayserver.org/Deployment/).
+
+### 🧪 API
+
+BTCPay Server has two API:
```
Comment (Kukks) [inline README.md:197]:
```suggestion
BTCPay Server has two APIs:
```

**Review by Kukks — CHANGES_REQUESTED** [review]
small nits

File: README.md (hunk 1 above)
Comment (pavlenex) [reply README.md:8]: It's mentioned in the features. And in a way doesn't have to be self-hosted. But the main reason I didn't put it is the "self hosted" wallet drama. If you think it's important I can add it.

File: README.md (hunk 4 above)
Comment (pavlenex) [reply README.md:120]: Master community chat!

Comment (pavlenex) [reply README.md]: Actually, since we now have community section I removed this sentence as it was unrelated to documentation section. d887546e5834451bbc6d4993d1553e97516e296a

**Review by dennisreimann — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,98 +1,143 @@
...
+* Share your instance with friends (multi-tenant)
+* Invoice management and Payment requests
+* Apps: Point of sale, crowdfunding, donation button
+* Full-node reliant wallet with [hardware wallet integration](https://docs.btcpayserver.org/Vault/) and SegWit support
+* Bitcoin-only build, separate community-maintained altcoin build ([supported altcoins](https://docs.btcpayserver.org/FAQ/FAQ-Altcoin/))
 
-## Getting Started
+## 🚀 Getting Started
 
 Firstly, decide if you want to host an instance yourself or use a [third-party host](https://docs.btcpayserver.org/ThirdPartyHosting/). If you've chosen to self-host, we documented plenty [ways to deploy BTCPay Server](https://docs.btcpayserver.org/Deployment/).
```
Comment (Zaxounette) [inline README.md:108]:
```suggestion
Firstly, decide if you want to host an instance yourself or use a [third-party host](https://docs.btcpayserver.org/ThirdPartyHosting/). If you've chosen to self-host, we documented plenty of [ways to deploy BTCPay Server](https://docs.btcpayserver.org/Deployment/).
```

Old un-noticed typo

File: README.md [hunk 7]
```diff
@@ -1,98 +1,143 @@
...
 
 After successful deployment, make sure to check our [getting started](https://docs.btcpayserver.org/RegisterAccount/) and [walkthrough](https://docs.btcpayserver.org/Walkthrough/) guides. In case you would like to use Lightning Network, see [Lightning guide](https://docs.btcpayserver.org/LightningNetwork/).
 
-## Documentation
+## 📗 Documentation
 
 Please check out our [official website](https://btcpayserver.org/), our [complete documentation](https://docs.btcpayserver.org/) and [FAQ](https://docs.btcpayserver.org/FAQ/) for more details.
```
Comment (Zaxounette) [inline README.md:114]:
```suggestion
Please check out our [official website](https://btcpayserver.org/), [complete documentation](https://docs.btcpayserver.org/) and [FAQ](https://docs.btcpayserver.org/FAQ/) for more details.
```

Repetition.

File: README.md [hunk 8]
```diff
@@ -1,98 +1,143 @@
...
 
-If you have trouble using BTCPay, consider joining [communities listed on official website](https://btcpayserver.org/#communityCTA) to get help from BTCPay community members. Only file [Github issue](https://github.com/btcpayserver/btcpayserver/issues) for technical issues you can't resolve through other channels or feature requests you've validated with other members of community.
-
-Main community chat is located on [Mattermost](https://chat.btcpayserver.org/).
+If you have trouble using BTCPay, consider joining [communities listed on the official website](https://btcpayserver.org/#communityCTA) to get help from the BTCPay community members. Only file [Github issue](https://github.com/btcpayserver/btcpayserver/issues) for technical issues you can't resolve through other channels or feature requests you've validated with other members of the community.
```
Comment (Zaxounette) [inline README.md:116]:
```suggestion
If you have trouble using BTCPay Server, consider joining [communities listed on the official website](https://btcpayserver.org/#communityCTA) to get help from the members. Only file [Github issue](https://github.com/btcpayserver/btcpayserver/issues/new/choose) for technical issues you can't resolve through other channels or feature requests you've validated with other members of the community.
```

* BTCPay Server for consistency
* Update the `github new issue` link,
* Delete word repetition (`community`)

**Comment by Zaxounette** [comment]
I'm taking this PR as an occasion to have a thorough read and review. Please stand by.

File: README.md [hunk 9]
```diff
@@ -1,98 +1,143 @@
...
+Contributors looking to do something a bit more challenging, before opening a pull request, please join [our community chat](https://chat.btcpayserver.org/) or [start a GitHub discussion](https://github.com/btcpayserver/btcpayserver/discussions) to get early feedback, discuss best ways to tackle the problem and ensure there is no work duplication.
+
+There are many other ways to get involved with the project. Check our [contribution guidelines](https://docs.btcpayserver.org/Contribute/). To get the big-picture of the project development, visit our [evolving roadmap](https://github.com/orgs/btcpayserver/projects/9). We organize different types of community calls several times a month. [Subscribe to our calendar](https://github.com/btcpayserver/organization#calendar) and feel free to join our calls to discuss development, design, documentation and more.
+
+## 🧑‍💻 Developing
+
+To begin developing locally, visit [local development guide](https://docs.btcpayserver.org/LocalDevelopment/). There are also several video-tutorials:
```
Comment (Zaxounette) [inline README.md:130]:
```suggestion
To begin developing locally, visit our [local development guide](https://docs.btcpayserver.org/LocalDevelopment/). There are also several video-tutorials:
```
Grammar police.

File: README.md [hunk 10]
```diff
@@ -1,98 +1,143 @@
...
-Here is some info about [how to extend the themes](https://docs.btcpayserver.org/Theme/).
+### How to build
 
-## How to build
+While the documentation advises using docker-compose, you may want to build BTCPay Server yourself.
 
-While the documentation advises to use docker-compose, you may want to build BTCPay Server yourself.
-
-First install .NET Core SDK v3.1 as specified by [Microsoft website](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+First, install .NET Core SDK v3.1 as specified by [Microsoft website](https://dotnet.microsoft.com/download/dotnet-core/3.1).
```
Comment (Zaxounette) [inline README.md:140]:
```suggestion
First, install .NET Core SDK v3.1 as specified by the [Microsoft website](https://dotnet.microsoft.com/download/dotnet-core/3.1).
```
Grammar police.

File: README.md [hunk 11]
```diff
@@ -136,23 +181,34 @@ If you need to debug ledger wallet interaction, install the development time cer
...
-## Other dependencies
+### Other dependencies
 
 For more information, see the documentation:
-[How to deploy a BTCPay server instance](https://docs.btcpayserver.org/Deployment/).
+[How to deploy a BTCPay Server instance](https://docs.btcpayserver.org/Deployment/).
+
+### 🧪 API
+
+BTCPay Server has two APIs:
```
Comment (Zaxounette) [inline README.md:193]:
```suggestion
BTCPay Server has two API's:
```
Grammar'ish police.

File: README.md [hunk 12]
```diff
@@ -136,23 +181,34 @@ If you need to debug ledger wallet interaction, install the development time cer
...
+
+- **Greenfield API (New)**
+  - [Greenfield API documentation](https://docs.btcpayserver.org/API/Greenfield/v1/)
+  - [Greenfield API examples with CURL](https://docs.btcpayserver.org/GreenFieldExample/)
+- **Legacy API**
+
+The **Greenfield API** is our brand-new API which is still in development. Once complete, it will allow you to run BTCPay Server heedlessly.
+The **legacy API**, is fully compatible with [BitPay's API](https://bitpay.com/api/). It has limited features, but allows instant migration from BitPay.
```
Comment (Zaxounette) [inline README.md:201]:
```suggestion
The **Legacy API**, is fully compatible with [BitPay's API](https://bitpay.com/api/). It has limited features, but allows instant migration from BitPay.
```
Capital L.

File: README.md [hunk 13]
```diff
@@ -136,23 +181,34 @@ If you need to debug ledger wallet interaction, install the development time cer
...
 
-## Supported altcoins
+## 💚 Community
 
-Bitcoin is the only focus of the project and its core developers. However, opt in integrations are present for [several altcoins](https://docs.btcpayserver.org/FAQ/FAQ-Altcoin/). Altcoins are maintained by their respective communities.
+Our community is ❤️ of the project. To chat with other community members in real-time, join our [Mattermost chat](https://chat.btcpayserver.org). We're also on [GitHub discussions](https://github.com/btcpayserver/btcpayserver/discussions).
```
Comment (Zaxounette) [inline README.md:205]:
```suggestion
Our community is the ❤️ of the project. To chat with other community members in real-time, join our [Mattermost chat](https://chat.btcpayserver.org). We're also on [GitHub discussions](https://github.com/btcpayserver/btcpayserver/discussions).
```
Grammar police.

**Review by Zaxounette — CHANGES_REQUESTED** [review]
Just grammar police nitpicks (I'm @bolatovumar 's son on that one) and a few old paragraph reviews that can be changed while this file is being worked on.

**Review by NicolasDorier — APPROVED** [review]

File: README.md [hunk 14]
```diff
@@ -1,98 +1,143 @@
...
+## 📗 Documentation
 
-If you have trouble using BTCPay, consider joining [communities listed on official website](https://btcpayserver.org/#communityCTA) to get help from BTCPay community members. Only file [Github issue](https://github.com/btcpayserver/btcpayserver/issues) for technical issues you can't resolve through other channels or feature requests you've validated with other members of community.
+Please check out our [official website](https://btcpayserver.org/), [complete documentation](https://docs.btcpayserver.org/) and [FAQ](https://docs.btcpayserver.org/FAQ/) for more details.
 
-Main community chat is located on [Mattermost](https://chat.btcpayserver.org/).
+If you have trouble using BTCPay Server, consider joining [communities listed on the official website](https://btcpayserver.org/#communityCTA) to get help from the members. Only file [Github issue](https://github.com/btcpayserver/btcpayserver/issues/new/choose) for technical issues you can't resolve through other channels or feature requests you've validated with other members of the community.
```
Comment (britttttk) [inline README.md:116]:
```suggestion
If you have trouble using BTCPay Server, consider joining [communities listed on the official website](https://btcpayserver.org/#communityCTA) to get help from other contributors. Only create a [GitHub issue](https://github.com/btcpayserver/btcpayserver/issues/new/choose) for technical issues you can't resolve through other channels or feature requests you've validated with other members of the community.
```

File: README.md [hunk 15]
```diff
@@ -136,23 +181,34 @@ If you need to debug ledger wallet interaction, install the development time cer
...
+### 🧪 API
+
+BTCPay Server has two API's:
+
+- **Greenfield API (New)**
+  - [Greenfield API documentation](https://docs.btcpayserver.org/API/Greenfield/v1/)
+  - [Greenfield API examples with CURL](https://docs.btcpayserver.org/GreenFieldExample/)
+- **Legacy API**
+
+The **Greenfield API** is our brand-new API which is still in development. Once complete, it will allow you to run BTCPay Server heedlessly.
```
Comment (britttttk) [inline README.md:200]:
```suggestion
The **Greenfield API** is our brand-new API which is still in development. Once complete, it will allow you to run BTCPay Server headlessly.
```

**Review by britttttk — CHANGES_REQUESTED** [review]
looks great @pavlenex! definitely needed a refresh
